README.md

```diff
@@ -15,6 +15,75 @@ If you can't find what you are looking for, try the [old docs on the ODK website
 
 The following steps help you build and view ODK docs locally. For details on each of these steps, see the [Contributor Guide](http://docs.opendatakit.org/contributing).
 
+### Prerequisites
+
+ * Install Docker
+   * Windows and Mac users should follow the instructions at [the get started guide](https://www.docker.com/get-started)
+   * Linux users should follow the instructions for their specific distribution: [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/), [Debian](https://docs.docker.com/install/linux/docker-ce/debian/), [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/), [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/), [Binaries](https://docs.docker.com/install/linux/docker-ce/binaries/)
+   
+   More info at the [Docker CE docs page](https://docs.docker.com/install/)
+ * Install [Git-LFS](https://git-lfs.github.com/).
+
+### Cloning the repo
```

Comment:
I think people should probably clone and fork like we do in the code repos. I like it how we do it on [Collect](https://github.com/opendatakit/collect#setting-up-your-development-environment), but I think that might be too much for this PR so I've filed an issue at https://github.com/opendatakit/docs/issues/880